### DISCUSSION TITLE
[Question] How to set multiple items in a PyDict simultaneously?

### DISCUSSION THREAD

**Discussion by rkshthrmsh** [PyO3/pyo3#3362] [Questions]
I have come across the `set_item` method for setting a single key-value pair in a `PyDict`. However, this quickly becomes many lines of code for setting multiple items. I was wondering if there is a way to set multiple items simultaneously when the key-value pairs are already known?

**Comment by davidhewitt** [comment]
We do not currently have an API for this; maybe #1703 might solve this eventually. I agree it can be tedious. You may also find [`pythonize`](https://github.com/davidhewitt/pythonize) useful to build Rust structs and turn them into Python dictionaries using `serde` machinery.
